### PR TITLE
db/main-readme

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added the main readme.

- [x]  Installation
- [x] Components
- [x] Working with Data
- [x] GUI
- [x] Edit Mode
- [x] Your First Dashboard
- [x] FAQ

### CODE REVIEW

File: ts/Dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+* Importing as a script
+```html
+    <script src="https://code.highcharts.com/dashboards.js"></script>
+ ```
+
+To fully utilize the Dashboards potential, there might be a need to load additional modules. <br>
+In the [documentation](https://highcharts.com/docs/dashboards/instalation), you can find more information on how to do that and what are the other ways of importing the Dashboards.
+
+## Components
+Each dashboard is built with different components. You can add the most basic HTMLComponent where you can add some text, image etc.
```
Comment (sebastianbochan) [inline ts/Dashboard/README.md:20]: Can we wrap names like HTMLComponent into `HTMLComponent`? It will increase readibility

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by pawellysy — CHANGES_REQUESTED** [review]
FAQ?

File: ts/Dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                }]
+            }]
+        }]
+    }
+```
+More about the GUI in the [documentation](https://highcharts.com/docs/dashboards/gui).
+
+
+## Edit Mode
+The Edit Mode is a part of the dashboard that allows you to edit the dashboard. You can add, remove and edit components. <br>
```
Comment (bre1470) [inline ts/Dashboard/README.md:66]: There is not need for `<br>`. You can add double space at the end to create a paragraph.

File: ts/Dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+To enable the Edit Mode you have to add the `highcharts-dashboard` class to the div where the dashboard is placed. <br>
```
Comment (bre1470) [inline ts/Dashboard/README.md:67]: = `highcharts-dashboards`

File: ts/Dashboard/README.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+More about the Edit Mode in the [documentation](https://highcharts.com/docs/dashboards/edit-mode).
```
Comment (bre1470) [inline ts/Dashboard/README.md:69]: The links are not a11y-friendly. The text link needs to be more specific. `You find more information in the [Edit Mode documentation](…` for e.g. helps more than multiple links saying "documentation".

**Review by bre1470 — CHANGES_REQUESTED** [review]
Good job! 👍 Just some adjustments...

**Review by sebastianbochan — APPROVED** [review]

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Pull request merged**